### DISCUSSION TITLE
How do I start Far Manager from PowerShell?

### DISCUSSION THREAD

**Discussion by jdrch** [FarGroup/FarManager#766] [General]
Just wondering the above. I installed the program using the .msi GUI installer, with the Path feature selected. I then restarted Windows terminal to pick up the new PATH variables. 

Googling shows people using the `far` command, but when I try that I get an error message:

```
> far
far: The term 'far' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Any ideas?

**Comment by yegor-mialyk** [comment]
Use a full path to Far.exe as when you open Powershell it does not inherit the environment's %PATH% variable.
Or you can create a 'far' alias and point it to a full path (`Set-Alias`). Or update the PATH environment variable yourself.

**Reply by jdrch** [reply]
That worked, thanks!
